models/aave/optimism/aave_v3_optimism_flashloans.sql

```diff
@@ -0,0 +1,49 @@
+{{ config(
+    schema = 'aave_v3_optimism'
+    , alias='flashloans'
+    , post_hook='{{ expose_spells(\'["optimism"]\',
+                                  "project",
+                                  "aave_v3",
+                                  \'["hildobby"]\') }}'
+  )
+}}
+
+{% set aave_mock_address = '0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee' %}
+{% set wmatic_address = '0x0d500b1d8e8ef31e21c99d1db9a6444d3adf1270' %}
```

Comment:
copy/paste error? not supposed to be matic throughout script?

Reply:
ah yes indeed, good catch, just fixed!